network_params.yaml

```diff
@@ -44,6 +47,8 @@ optimism_package:
     image: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-deployer:v0.0.12
     l1_artifacts_locator: https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-fffcbb0ebf7f83311791534a41e65ef90df47797f9ca8f86941452f597f7128c.tar.gz
     l2_artifacts_locator: https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-fffcbb0ebf7f83311791534a41e65ef90df47797f9ca8f86941452f597f7128c.tar.gz
+  observability:
+    enabled: true
```

Comment:
Why this change?

Reply:
Trying to make the network_params.yaml more explicit in terms of which services are optional (have an enabled flag).
devnet is pretty slow to spin up, so when I don't need certain components, I want to be able to make them disabled. It's nice when the example network_params.yaml file shows which components can be disabled out of the box.
For eg this way a newcomer can see directly whether that they can disable observability and challenger, but not the batcher for eg.

Reply:
Oh yes agreed, what I mean is that the PR aims to make a challenger change - I would keep PRs clean in terms of changes and submit any changes not related to the main goal of the PR as separate PRs

Reply:
[b56365f](https://github.com/ethpandaops/optimism-package/pull/165/commits/b56365ff25c38aae9eb49e405f7c983cee33f966)